packages/gasket-plugin-docusaurus/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "@gasket/plugin-docusaurus",
+  "version": "6.21.0",
+  "description": "Gasket plugin for docusaurus",
+  "main": "lib",
+  "files": [
+    "lib"
+  ],
+  "scripts": {
+    "lint": "eslint .",
+    "lint:fix": "npm run lint -- --fix",
+    "test": "npm run test:runner",
+    "test:runner": "mocha test/*.test.js --require test/setup.js",
+    "test:watch": "npm run test:runner -- --watch",
+    "test:coverage": "nyc --reporter=text --reporter=json-summary npm run test:runner",
+    "posttest": "npm run lint",
+    "report": "nyc report --reporter=lcov"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/godaddy/gasket.git"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "keywords": [
+    "docs",
+    "viewer",
+    "gasket",
+    "plugin"
+  ],
+  "author": "GoDaddy Operating Company, LLC",
+  "maintainers": [
+    "Michael Mason <mmason2@godaddy.com>"
+  ],
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/godaddy/gasket/issues"
+  },
+  "homepage": "https://github.com/godaddy/gasket/tree/main/packages/",
+  "dependencies": {
```

Comment:
You are using `lodash.defaultsdeep` in your code, so you should include this as a direct dependency in the package.json as well. It is probably is working since the doc's plugin brings it in, but it would be safer to explicitly depend on it in this package, incase the docs plugin's dependencies ever change.
```
"lodash.defaultsdeep": "^4.6.1",
```